Pachanga/Core/Firestore/MatchManager.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  MatchManager.swift
+//  Pachanga
+//
+//  Created by Javier Alaves on 21/8/23.
+//
+
+import FirebaseFirestore
+import FirebaseFirestoreSwift
+import Foundation
+import SwiftUI
+
+struct Match: Codable {
+    
+    // props
+    let matchId: String
+    let dateCreated: Date
+    var location: String
+    var matchDate: Date
+    var players: [String] // combo of teamOne and teamTwo
+    var teamOne: [String] // tuple instead?
```

Comment:
Yeah, tuples would be much better since Arrays store data dynamically (costlier) and tuples have a fixed size of 2, making them much more efficient to query